Utilities/bootstrap

```diff
@@ -126,24 +130,51 @@ class Target(object):
     def __init__(self, name, dependencies=[], swiftflags=[], extra_libs=[],
                  subpath=None):
         self.name = name
+        # FIXME: This isn't proper c99 name.
```

Comment:
"Meh", doesn't even need to be a FIXME, bootstrap just has to get us past stage1.